integration/service/oasis/solution/status_code.go

```diff
@@ -0,0 +1,27 @@
+package solution
+
+const (
```

Comment:
- comment; 
- there's a risk here that if you add another const between current twos, the original value will be changed that will cause compatible issue. Better to expicitly either write down the codes or use `string` instead of `int` as status code type.

Reply:
Very good suggestions.  Will use pre-defined code.